course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,103 @@
+
+# Lecture schedule
+| Week | Date   | Topic                                               | Instructor     |
+|------|--------|-----------------------------------------------------|----------------|
+| 1    | Sep 07 | Welcome to the course! Intro to programming         | Everyone       |
+| 2    | Sep 12 | Basic R – vectors, data frames                      | Joel           |
+| 3    | Sep 14 | Basic R – scripting, conditionals, loops            | Joel           |
+| 4    | Sep 19 | Intro to dplyr                                      | Joel           |
+| 5    | Sep 21 | More dplyr                                          | Joel           |
+| 6    | Sep 26 | Qualitative review of population models             | Madeleine      |
+| 7    | Sep 28 | Qualitative population models in 2 dimensions       | Madeleine      |
+| 8    | Oct 03 | Fixed points, null clines, and stability            | Madeleine      |
+| 9    | Oct 05 | Modeling in R                                       | Madeleine      |
+| 10   | Oct 10 | Fitting models to data                              | Madeleine      |
+| 11   | Oct 12 | Linear regression population dynamics models?       | Luke           |
+| 12   | Oct 17 | Introduce data sets, hypotheses, start projects     | Luke           |
+| 13   | Oct 19 | Project work, prodigenR, tidy data, git via rstudio | Luke           |
+| 14   | Oct 24 | Data visualization in R                             | Lindsay        |
+| 15   | Oct 26 | Project work, data visualizations                   | Lindsay        |
+| 16   | Oct 31 | Reproducible workflow, metadata, Rmarkdown          | Lina           |
+| 17   | Nov 02 | Project work, metadata                              | 1 TA/group     |
+| -    | Nov 07 | Fall break                                          | -              |
+| -    | Nov 09 | Fall break                                          | -              |
+| 18   | Nov 14 | Scientific collaboration, Git and GitHub            | Elliot         |
+| 19   | Nov 16 | Project work                                        | 1 TA/group     |
+| 20   | Nov 21 | Preparing a scientific item for academic publishing | Lina           |
+| 21   | Nov 23 | Project work                                        | 1 TA/group     |
+| 22   | Nov 28 | Project work                                        | 1 TA/group     |
+| 23   | Nov 30 | Project work                                        | Everyone       |
+| 24   | Dec 05 | Project work                                        | Everyone       |
+
+# Assessment schedule 
+| Assesment             | Type                | Due date   | Marks |
+|-----------------------|---------------------|------------|-------|
+| Getting set up        | Individual          | Sep 11     | 4     |
```

Comment:
The dates are chosen so that each assignment is due on Monday the following week. Is this enough time if we hand out assignments during the Thursday lecture? I chose Monday since it would give us some time to briefly go over most of the assignments before the next class and briefly repeat general concepts if many students struggled with the same problem.

Thinking about it a little more, I believe a due date on Tuesday, might be better. This gives students a chance to ask questions before and after class regarding the assignment, before they hand it in. It would fit particularly well if we make Tuesdays our office hours. It would also give us an additional day (wednesday) to go over the assignments and bring up and key concept on the following Thursday lecture. I will change to this in the next iteration unless there are opposing views.